src/src/PluginStructs/P016_data_struct.cpp

```diff
@@ -159,18 +154,32 @@ void P016_data_struct::loadCommandLines(struct EventStruct *event) {
 }
 
 void P016_data_struct::saveCommandLines(struct EventStruct *event) {
-  int saveOffset = 0;
-
   for (uint8_t i = 0; i < P16_Nlines; i++) {
-    SaveToFile(SettingsType::Enum::CustomTaskSettings_Type,
-               event->TaskIndex,
-               reinterpret_cast<const uint8_t *>(&(CommandLines[i])),
-               sizeof(tCommandLinesV2),
-               saveOffset);
-    saveOffset += sizeof(tCommandLinesV2);
+    saveCommandLine(event, CommandLines[i], i);
   }
 }
 
+void P016_data_struct::loadCommandLine(struct EventStruct *event, tCommandLinesV2 &line, uint8_t lineNr)
+{
+  const int loadOffset = lineNr * sizeof(tCommandLinesV2);
+  LoadFromFile(SettingsType::Enum::CustomTaskSettings_Type,
```

Comment:
You might need to check for the V1 settings, and read & convert the old data format, see `convertCommandLines`, as implemented when calling `loadCommandLines`, to ensure backward compatibility.

Reply:
Almost forgot this part...
Is now pushed too :)